cdc/redo/manager.go

```diff
@@ -208,8 +201,8 @@ func NewManager(ctx context.Context, cfg *config.ConsistentConfig, opts *Manager
 	}
 
 	if opts.EnableBgRunner {
```

Comment:
when EnableBgRunner = false ?

Reply:
why not merge the two bg method ?

Reply:
> when EnableBgRunner = false ?

In owner, ref: https://github.com/pingcap/tiflow/blob/master/cdc/owner/changefeed.go#L379


> why not merge the two bg method ?

Good advice!